### DISCUSSION TITLE
Problem Import csv with quotes

### DISCUSSION THREAD

**Discussion by bjc49** [RafaelGB/obsidian-db-folder#742] [Q&A]
Hello
I just tried to create a database from a csv file import generated by another application (Gramps - Genealogy)
This file contains quotation marks as the csv format allows it I believe?
The commas that are between these quotes should not be interpreted as a separator if I understood correctly?
This is not the case with this "Import csv" function in Db Folder resulting in an inconsistent generated table
There may be a solution?
Thanks
Jean Claude

`File,Nom ,Genre,Naissance,Lieu de naissance,Décès,Lieu de décès,Conjoint ,Étiquettes,Modifié
I0010,"Brévard, Anne",féminin,21/12/1787,"Verron, La Flèche, Sarthe",27/11/1835,"Le Mans, Sarthe","Duchesne, François",Recherche,29/9/2022 06:27:52`

**Comment by RafaelGB** [comment]
Hello @bjc49 ,

yeah, there is a reported problem about https://github.com/RafaelGB/obsidian-db-folder/issues/734
